x-pack/test/functional/apps/maps/index.js

```diff
@@ -27,15 +27,17 @@ export default function ({ loadTestFile, getService }) {
       await esArchiver.unload('maps/kibana');
     });
 
-    describe('', function () {
+    // FF: suite fails in headless mode
+    describe.skip('', function () {
```

Comment:
Hey sir, I see the block was already not named, so my question is orthogonal to this pr...but why is it not named?

Reply:
We use `describe` in `index` files to organize test files in CI groups per design - we are shifting files from time to time to improve execution speed. It can contain pretty different tests, so naming does not make much sense here. Still not the best design.